numpy/linalg/umath_linalg.c.src

```diff
@@ -1907,9 +1816,10 @@ dump_geev_params(const char *name, GEEV_PARAMS_t* params)
    #TYPE = FLOAT, DOUBLE#
    #CTYPE = CFLOAT, CDOUBLE#
    #typ = float, double#
-   #complextyp = COMPLEX_t, DOUBLECOMPLEX_t#
+   #complextyp = npy_cfloat, npy_cdouble#
    #lapack_func = sgeev, dgeev#
    #zero = 0.0f, 0.0#
+   #suffix = f,  #
```

Comment:
Do you need both `zero` and `suffix`? I see you changed a `@zero@` below to use `@suffix@`. I don't really care which you use.